app/main.py

```diff
@@ -1,3 +1,54 @@
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        print(f"Distance: {self.km} kilometers.")
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, int):
+            return Distance(self.km + other)
+
+    def __iadd__(self, other):
+        if isinstance(other, int):
+            self.km += other
+        else:
+            self.km += other.km
+        return self
+
+    def __mul__(self, other):
+
+        if isinstance(other, int):
+            return Distance(self.km * other)
+        else:
+            return Distance(self.km * other.km)
```

Comment:
`other` can't be an instance of `Distance`, only a number (float or int)
Because if you multiply a kilometer by a kilometer, you get a square kilometer